docs/install_guides/installing-openvino-from-archive-macos.md

```diff
@@ -93,6 +93,7 @@ You must update several environment variables before you can compile and run Ope
 
 .. code-block:: sh
 
+   cd /opt/intel/openvino_2023 
```

Comment:
it's not required since you use absolute paths below.

Reply:
ok, but can we fix the original issue?

Reply:
We could check the reason of this behavior - I have observed this on MacOS 10.15 only (It does not reproduces on MacOS 12), so it seems that it is OS related. But I will agree - the best solution would be to provide fix for setupvars.sh